rest/src/main/java/org/jboss/pnc/rest/endpoints/ArtifactEndpointImpl.java

```diff
@@ -75,6 +79,20 @@ public void init() {
                 Optional.ofNullable(sha1));
     }
 
+    @Override
+    public Page<ArtifactInfo> getAllFiltered(
+            PaginationParameters paginationParameters,
+            String identifier,
+            Set<ArtifactQuality> qualities,
+            RepositoryType repoType) {
+        return artifactProvider.getAllFiltered(
+                paginationParameters.getPageIndex(),
+                paginationParameters.getPageSize(),
+                Optional.ofNullable(identifier),
+                Optional.ofNullable(qualities),
```

Comment:
No need to wrap set to Optional:
> If `@DefaultValue` is not used and the corresponding meta-data is not present in the request, the value will be an empty collection for List, Set or SortedSet, null for other object types, and the Java-defined default for primitive types.
https://docs.oracle.com/javaee/7/api/javax/ws/rs/DefaultValue.html